src/Sql/dbo/Stored Procedures/User_DeleteById.sql

```diff
@@ -1,5 +1,5 @@
-﻿CREATE PROCEDURE [dbo].[User_DeleteById]
-    @Id UNIQUEIDENTIFIER
+﻿CREATE PROCEDURE [dbo].[User_DeleteByIds]
+    @Ids [dbo].[GuidIdArray]
```

Comment:
We're no longer using `GuidIdArray`, instead we parse a json string. Example: https://github.com/bitwarden/server/blob/main/src/Sql/dbo/Stored%20Procedures/User_ReadByIdsWithCalculatedPremium.sql